pkg/kubeconfig/create_kubecfg.go

```diff
@@ -104,30 +110,29 @@ func BuildKubecfg(cluster *kops.Cluster, keyStore fi.Keystore, secretStore fi.Se
 		}
 	}
 
-	if admin {
-		{
-			cert, key, _, err := keyStore.FindKeypair("kubecfg")
-			if err != nil {
-				return nil, fmt.Errorf("error fetching kubecfg keypair: %v", err)
-			}
-			if cert != nil {
-				b.ClientCert, err = cert.AsBytes()
-				if err != nil {
-					return nil, err
-				}
-			} else {
-				return nil, fmt.Errorf("cannot find kubecfg certificate")
-			}
-			if key != nil {
-				b.ClientKey, err = key.AsBytes()
-				if err != nil {
-					return nil, err
-				}
-			} else {
-				return nil, fmt.Errorf("cannot find kubecfg key")
-			}
+	if admin != 0 {
+		req := pki.IssueCertRequest{
+			Signer: fi.CertificateIDCA,
+			Type:   "client",
+			Subject: pkix.Name{
+				CommonName:   "kubecfg",
```

Comment:
Perhaps use `user.Current().Username` instead?

Reply:
Yeah, that's not a bad change to make at the same time.  Or maybe "admin-kubecfg-${USER}", just in case of usage inside a container where I expect most of these will end up as "root" or "jenkins" which might be confusing.